modules/custom/moj_resources/src/SeriesContentApiClass.php

```diff
@@ -205,9 +236,9 @@ private function getSeriesContentNodeIds($series_id, $number, $offset, $prison)
       ->condition('status', 1)
       ->accessCheck(false);
 
-    if ($series_id !== 0) {
+    // if ($series_id !== 0) {
```

Comment:
We should remove this instead of commenting out, ideally, if we intend to keep, would it be save to assume we should return a `400` response if there is no series ID?